src/chunk.c

```diff
@@ -612,24 +640,27 @@ get_am_name_for_rel(Oid relid)
  * instead needs the proper permissions on the database to create the schema.
  */
 Oid
-ts_chunk_create_table(Chunk *chunk, Hypertable *ht, char *tablespacename)
+ts_chunk_create_table(Chunk *chunk, Hypertable *ht, const char *tablespacename)
```

Comment:
Unfortunately, this doesn't work because none of the PostgreSQL functions or data types use const. We'd have to do a copy or cast in that case (for instance, the `makeRangeVar` below doesn't accept const).

Reply:
I missed that there were calls to PG functions in this function. I noticed in other similar place and didn't comment there.